framework/src/org/checkerframework/qualframework/poly/SimpleQualifierParameterAnnotationConverter.java

```diff
@@ -32,7 +33,7 @@
      * The default "Target" in an annotation is the primary qualifier.
      * We can't use null in the annotation, so we use this special value.
      */
-    public static final String PRIMARY_TARGET = "_primary";
+    public static final String PRIMARY_TARGET = DefaultValue.PRIMARY_TARGET;
```

Comment:
I think it would be clearer to move the javadoc from this field into DefaultValue and then remove this field.